.eslintrc.js

```diff
@@ -48,6 +48,8 @@ module.exports = {
           },
         ],
         'import/no-unresolved': 'off',
+        // TypeScript takes care of this for us (https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/namespace.md)
```

Comment:
is there a need to turn this off?

Reply:
I could turn it back on, but the import plugin (and specifcally the default node resolver) notoriously isn't able to resolve modern package structures using `export` conditions.  v6 of TS-eslint is the first version where they shift exclusively to using package export conditions for their api.  So that rule is going to have a hard time determining what's valid and what isn't, which is something TS is already doing for us.  I was having issues with it misfiring, and since it's redundant, I thought it was best to just turn it off.  What do you think?

Reply:
ahh all good, that's a good reason to disable it yeah